src/gt4py/backend/templates/stencil_module.py.in

```diff
@@ -151,7 +151,7 @@ class {{ class_name }}(StencilObject):
             exec_info["origin"] = _origin_
             exec_info["run_start_time"] = time.perf_counter()
 {%- filter indent(width=8) %}
-{{- implementation }}
+{{ implementation }}
```

Comment:
This change allows us to not have to manually add the newline before the implementation before "with"
```python
        sources.append(
            "with np.errstate(divide='ignore', over='ignore', under='ignore', invalid='ignore'):"
        )
```